lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -225,6 +229,14 @@ export class ClientAuthError extends AuthError {
             `${ClientAuthErrorMessage.endpointResolutionError.desc} Detail: ${errDetail}`);
     }
 
+    /**
+     * Creates an error thrown when the fetch client throws
+     */
+    static createNetworkError(endpoint: string, errDetail: string): ClientAuthError {
+        return new ClientAuthError(ClientAuthErrorMessage.networkError.code,
+            `${ClientAuthErrorMessage.networkError.desc} Detail: ${errDetail} Attempted to reach: ${endpoint}`);
```

Comment:
the `endpoint` part might be considered PII if it has something like loginHint in it.

Reply:
Right. This should just be the token endpoint which doesnt have a query string, but it may be worth stripping that off just in case

Reply:
Ah yeah that's true. Probably worth stripping off for now (or truncating it?)

Reply:
Done